presentation/src/main/kotlin/com/jshvarts/conductormvp/notes/NotesView.kt

```diff
@@ -38,9 +40,11 @@ class NotesView : BaseView(), NotesContract.View {
 
     override fun onAttach(view: View) {
         super.onAttach(view)
-        initRecyclerView(view.context)
-        presenter.start(this)
-        presenter.loadNotes()
+        recyclerView.initRecyclerView(LinearLayoutManager(view.context), recyclerViewAdapter)
+        with(presenter) {
+            presenter.start(this@NotesView)
```

Comment:
do you need to explicitly use `presenter` here like this? Doesn't the with give you `T.()`?

Reply:
that was the reason I added `with()` but forgot to remove `presenter` :D